schemas/rum/view-schema.json

```diff
@@ -281,6 +281,24 @@
               "type": "number",
               "description": "Minimum refresh rate during the view’s lifetime (in frames per second)",
               "readOnly": true
+            },
+            "vitals": {
```

Comment:
AFAIK, `vitals` has been introduced by Google as a well defined metrics selection as well as threshold levels to deterministically evaluate the user experience of an application. Vitals should be reported as a single number easy to evaluate.

What you are sending are simple metrics. Calling them `vitals`may be confusing as it does not really match the same concept.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I would suggest to treat them as other View metrics. As such, I would not nest them inside their own namepsace but add theme directly as `@view.*` attributes. You could still report multiple values (min/max/average). Example:

```json
{
  "view": {
    ...
    "flutter_build_time": {
      "min": 123,
      "max": 123,
      "avg": 123,
    }
}
```

Reply:
That might be a bigger conversation with @priyanshi-gupta -- we've been referring to these extra metrics as "Additional Mobile Vitals" but if we want to change the name to not confuse with Google's naming, I'm okay with that. Although "performance metrics" also has some baggage...

The hope with putting them into their own namespace was to (potentially) allow the front end to pick up these metrics and display them as "vitals" without having to know too much about them. Basically, anything in that namespace is assumed to be of the same type and displayed in the same way, queried the same way, etc. 

I'm okay moving them out of a namespace if it's not useful... just thought it might be.